src/target/riscv/riscv.c

```diff
@@ -4501,6 +4583,27 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Execute a sequence of 32-bit instructions using the program buffer. "
 			"The final ebreak instruction is added automatically, if needed."
 	},
+	{
+		.name = "equality_match_trigger_enable",
+		.handler = riscv_equality_match_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use equality match trigger in bp/wp"
+	},
+	{
+		.name = "napot_trigger_enable",
+		.handler = riscv_napot_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use NAPOT trigger in bp/wp"
+	},
+	{
+		.name = "ge_lt_trigger_enable",
+		.handler = riscv_ge_lt_trigger_enable,
+		.mode = COMMAND_CONFIG,
+		.usage = "[on|off]",
+		.help = "Allow OpenOCD to use GE/LT triggers in bp/wp"
```

Comment:
There is inconsistency between the help and the implementation: The help say `Allow OpenOCD to use ... in bp/wp` but only watchpoints are affected. 

What is your intention? To affect just watchpoints or also breakpoints?

Reply:
There was only one general function for setting trigger for wp and bp, so this options effected on wp and bp. Later I split this function info two (one for wp and one for bp) and forgot to change help message.
Addressed

Reply:
Thanks.